itenium-socks/src/app/app.component.ts

```diff
@@ -1,4 +1,4 @@
-import { Component } from '@angular/core';
+import { Component, LOCALE_ID } from '@angular/core';
```

Comment:
You have added a reference that is not used!

Always go through your own PR to see that everything you are about to commit makes sense.
Give yourself the opportunity to delete all changes you made that ended up not being useful/used!